### PR TITLE
Ability to set keyboardDisplayRequiresUserAction

### PR DESCRIPTION
**Pull request by craigblagg**
Issue:
When setting focus on an input that lives inside the webview, the keyboard is not displayed.
UIWebView offers keyboardDisplayRequiresUserAction which allows for this to act as one would expect.

Solution:
Use documented approaches to recreating this for a WKWebView - and expose it to React Native.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -70,12 +70,12 @@
   "readmeFilename": "README.md",
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/CRAlpha/react-native-wkwebview.git"
+    "url": "git+https://github.com/craigblagg/react-native-wkwebview.git"
```
Comment (chitezh) [inline package.json]: Please, revert these change so it can be merged

File: react-native-wkwebview.podspec [hunk 2]
```diff
@@ -8,12 +8,12 @@ Pod::Spec.new do |s|
   s.summary      = "React Native WKWebView for iOS"
   s.author       = "Ruoyu Sun <ruoysun@gmail.com> (https://github.com/insraq)"
 
-  s.homepage     = "https://github.com/CRAlpha/react-native-wkwebview"
+  s.homepage     = "https://github.com/craigblagg/react-native-wkwebview"
```
Comment (chitezh) [inline react-native-wkwebview.podspec]: this one too

**Comment by craigblagg** [comment]
Have now rebased the branch - dropping the changes for your comments above (had just updated them so could use my branch in our apps).

**Comment by craigblagg** [comment]
It’s the same swizzling approach that is in use within the Ionic framework
- with which a number of apps are in the App Store.

So don’t foresee concerns around that - but haven’t first hand put an app
in the App Store with it as yet.

**Comment by chitezh** [comment]
I can confirm that it’s not problematic during app review.
https://github.com/Telerik-Verified-Plugins/WKWebView/commit/04e8296adeb61f289f9c698045c19b62d080c7e3
https://stackoverflow.com/a/48623286/3297914

**Comment by craigblagg** [comment]
👍 I'll keep an eye out for any changes in the signature, and update accordingly should they come up

**Comment by brandons** [comment]
@craigblagg This solution is not working for me on iOS 11.4. The keyboard behavior does not change when adding the `keyboardDisplayRequiresUserAction` flag. Further adding the swizzle directly in my AppDelegate has no effect either.

**Comment by craigblagg** [comment]
I’m away from a machine for a week, but I’ll look at the method signature
when back - to see if it’s changed to. As that changing would be the only
thing to effect it.

**Comment by jordoh** [comment]
@brandons - this may not apply to 11.4, but 12.2+ has a different selector (`_elementDidFocus:userIsInteracting:blurPreviousNode:changingActivityState:userObject`) swizzled in react-native-webview's implementation: https://github.com/react-native-community/react-native-webview/pull/540/files#diff-cffebd9c9732dc4e4399daaa2f525bebR398
